frontend/src/metabase/public/components/EmbedFrame/LogoBadge.styled.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from "@emotion/styled";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { color } from "metabase/lib/colors";
+import { breakpointMinMedium } from "metabase/styled-components/theme";
+
+export const MetabaseLink = styled(ExternalLink)`
+  display: flex;
+  align-items: center;
+
+  font-size: 0.85rem;
```

Comment:
How do you convert `1.12em` (from `.h4`) to rem?
https://github.com/metabase/metabase/blob/62f05deaad7523eeb22f883f1bc5ee528802c092/frontend/src/metabase/css/core/headings.css#L31-L33

Reply:
Oh, that one used to work a little bit weirdly. We used to have [the `ExternalLink`](https://github.com/metabase/metabase/blob/2553505eb8775a8a90273b993fcdfbd194bd4bed/frontend/src/metabase/public/components/LogoBadge.jsx#L11) with `.h4` class, and [this `span` element](https://github.com/metabase/metabase/blob/2553505eb8775a8a90273b993fcdfbd194bd4bed/frontend/src/metabase/public/components/LogoBadge.jsx#L14) with `.text-small`. Both `h4` and `text-small` use `em` for setting the font size, so the font size is related to the parent element's font size. So the final font-size wasn't either `h4` or `text-small` one, but `text-small * h4 * whatever was set in h4's parent`. The final size was just about `0.85rem`, so I removed this implicitness with a single value here